app/src/main/java/io/github/sds100/keymapper/system/accessibility/MyAccessibilityService.kt

```diff
@@ -465,6 +557,54 @@ class MyAccessibilityService : AccessibilityService(), LifecycleOwner, IAccessib
         return Error.SdkVersionTooLow(Build.VERSION_CODES.N)
     }
 
+    override fun interactWithScreenElement(fullName: String, onlyIfVisible: Boolean, interactiontype: INTERACTIONTYPE, inputEventType: InputEventType): Result<*> {
+
+        Timber.d("interactWithScreenElement fullName: %s, onlyIfVisible: %s, interactionType: %s", fullName, onlyIfVisible.toString(), interactiontype.name)
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
+            if (rootInActiveWindow != null) {
+                // TODO: Find a way to get the "SystemView" as "rootInActiveWindow"
+                // Use a custom function because "findAccessibilityNodeInfosByViewId" does not iterate through all children sometimes
+                val nodeToInteractWith = findNodeByFullyQualifiedName(fullName, rootInActiveWindow)
```

Comment:
@sds100 if you have ANY IDEA if it's possible to get the "system view " on that, it would be super nice ... I mean not just within the current active window scope, I mean the absolute root view - you know what I mean?